--- a/system-config-printer-kde/cmake-modules/FindSystemConfigPrinter.py
+++ b/system-config-printer-kde/cmake-modules/FindSystemConfigPrinter.py
@@ -8,7 +8,7 @@ SYSTEM_CONFIG_PRINTER_DIR = "/usr/share/
 sys.path.append (SYSTEM_CONFIG_PRINTER_DIR)
 
 try:
-    import cupshelpers.ppds, cupshelpers.cupshelpers
+    import cupsutils.ppds, cupsutils.cupshelpers
 except:
     exit(1)
 print "Groovy"
--- a/system-config-printer-kde/system-config-printer-kde.py
+++ b/system-config-printer-kde/system-config-printer-kde.py
@@ -34,7 +34,7 @@ import thread
 
 #load modules from system-config-printer-common (debug, smburi), change path here if you have it installed elsewhere
 SYSTEM_CONFIG_PRINTER_DIR = "/usr/share/system-config-printer"
-if os.path.exists(SYSTEM_CONFIG_PRINTER_DIR + "/debug.py"):
+if os.path.exists(SYSTEM_CONFIG_PRINTER_DIR + "/config.py"):
     sys.path.append(SYSTEM_CONFIG_PRINTER_DIR)
 
 from PyQt4.QtCore import *
@@ -65,9 +65,10 @@ cups.require ("1.9.27")
 
 # These come from system-config-printer
 import config
-import cupshelpers  #, options
+import cupsutils.ppds
+from cupsutils import cupshelpers  #, options
 from smburi import SMBURI
-from debug import *
+from cupsutils.debug import *
 
 import dbus
 import dbus.mainloop.qt
@@ -1695,7 +1696,7 @@ class NewPrinterGUI(QDialog):
             c = cups.Connection ()
             debugprint ("Fetching PPDs")
             ppds_dict = c.getPPDs()
-            self.ppds_result = cupshelpers.ppds.PPDs(ppds_dict,
+            self.ppds_result = cupsutils.ppds.PPDs(ppds_dict,
                                                      language=language)
             debugprint ("Closing connection (PPDs)")
             del c
@@ -1921,7 +1922,7 @@ class NewPrinterGUI(QDialog):
                         ppddict = self.ppds.getInfoFromPPDName (ppdname)
                         make_model = ppddict['ppd-make-and-model']
                         (make, model) = \
-                            cupshelpers.ppds.ppdMakeModelSplit (make_model)
+                            cupsutils.ppds.ppdMakeModelSplit (make_model)
                         self.auto_make = make
                         self.auto_model = model
                 except:
@@ -2293,7 +2294,7 @@ class NewPrinterGUI(QDialog):
         if make_and_model and not device.id:
             mk = None
             md = None
-            (mk, md) = cupshelpers.ppds.ppdMakeModelSplit (make_and_model)
+            (mk, md) = cupsutils.ppds.ppdMakeModelSplit (make_and_model)
             device.id = "MFG:" + mk + ";MDL:" + md + ";DES:" + mk + " " + md + ";"
             device.id_dict = cupshelpers.parseDeviceID (device.id)
         # Check whether the device is supported by HPLIP and replace
